.env.example

```
@@ -0,0 +1,17 @@
+DB_PASSWORD=
+POSTGRES_HOST=
+POSTGRES_PORT=
+POSTGRES_DB=
+POSTGRES_USER=
+
+KEYCLOAK_CLIENT_SECRET=
+
+EUREKA_SERVER_URL=
+CONFIG_SERVER_URL=
+#EUREKA_HOST=
+#ZIPKIN_HOST=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)